Common/Common.h

```diff
@@ -12,6 +12,35 @@
 ///    See the License for the specific language governing permissions and
 ///    limitations under the License.
 
+#import <Security/AuthorizationPlugin.h>
+
+// Context & Hint Keys
+#define kAuthorizationEnvironmentAuthenticationAuthority \
```

Comment:
I'd suggest not using `kAuthorizationEnvironment*` as the beginning of each of these, as Apple could add the same names in the future.